erpnext/patches/v12_0/update_end_date_and_status_in_email.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.utils import add_days, getdate, today
+
+def execute():
+    email_campaign = frappe.get_all('Email Campaign')
+    if not email_campaign:
+        return
```

Comment:
No need of this. The for loop will take care of this as it will not enter the loop if there are no email campaigns found.